eng/pipelines/coreclr/crossgen2-hot-cold-splitting.yml

```diff
@@ -0,0 +1,59 @@
+trigger: none
```

Comment:
Adding a completely new pipeline would probably incur in unwanted additional costs. Just add the hot/cold splitting job to the current Crossgen2 pipeline (i.e. `eng/pipelines/coreclr/crossgen2.yml`